pom.xml

```diff
@@ -31,34 +26,44 @@
         <slf4j.version>1.7.31</slf4j.version>
         <querydsl.version>4.4.0</querydsl.version>
         <apt-maven-plugin.version>1.1.3</apt-maven-plugin.version>
+        <amazon-aws.version>1.12.21</amazon-aws.version>
+        <google-gson.version>2.8.7</google-gson.version>
         <jwt.version>3.17.0</jwt.version>
-        <json.version>20210307</json.version>
     </properties>
 
+    <parent>
+        <artifactId>spring-boot-starter-parent</artifactId>
+        <groupId>org.springframework.boot</groupId>
+        <version>2.3.12.RELEASE</version>
```

Comment:
`<version>${spring-boot.version}</version>`

Reply:
When I am trying to put that version through the "${}" in that parent I am catching an error in pom.xml.
![image](https://user-images.githubusercontent.com/85170274/126774834-7bb6be9a-954a-4a7a-929d-99ff562a19af.png)
But when I put the version directly, everything is fine.

Reply:
clear. go ahead